src/server.js

```diff
@@ -6,13 +6,15 @@ const dnsServer = dns.createServer();
 const https = require('https');
 const types = {
   1: 'A',
+  5: 'CNAME',
   16: 'TXT'
 };
 const API_BASE = ['', 'v1', 'dns'];
 const RECORD_TTL = 600;
 let records = {
   TXT: {},
-  A: {}
+  A: {},
+  CNAME: {},
```

Comment:
We just return based on the request type: https://github.com/fxbox/dns-server/blob/support-cname/src/server.js#L26  Without this change, we get undefined errors when assigning to a type not seen before: https://github.com/fxbox/dns-server/blob/master/src/server.js#L98